src/xrGame/PHCommander.h

```diff
@@ -73,6 +73,7 @@ class CPHCommander : public IPHWorldUpdateCallbck
 {
     Lock lock;
     PHCALL_STORAGE m_calls;
+    xr_unordered_map<CPHCall*, bool> m_callsUpdateDeferred;
```

Comment:
What about using xr_unique_ptr instead of raw pointer here?

Reply:
Hmm.. Look at [this](https://github.com/OpenXRay/xray-16/pull/340/files/84efb00c6866e6f5b4dba528c0df388e9b512440#diff-53994ec8d3d0fa54a96023d820d9de14R214). As I understand, it's impossible to get unique_ptr from raw pointer.